docs/application/web/get-started/wearable-watch/first-app-watch.md

```diff
@@ -162,7 +159,7 @@ Pay attention to the following main issues in the application source code, to un
     </widget>
     ```
 
-    > **Note**
+    > [!NOTE]
     >
```

Comment:
It looks like this additional > doesn't affect the layout of Note, but would you please check it again?

![image](https://user-images.githubusercontent.com/9958374/71965958-0918b300-3244-11ea-8bdd-33f0f2f491e0.png)